index.html

```diff
@@ -1,179 +1,67 @@
 <!doctype html>
 <html class="no-js" lang="en-US">
 <head>
-  <meta charset="utf-8">
-  <title>Scrim map generator</title>
-  <meta name="description" content="Generates map rotation for Destiny 2 scrims">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="theme-color" content="#fafafa">
-  <link rel="stylesheet" href="https://unpkg.com/spectre.css/dist/spectre.min.css">
-  <link rel="stylesheet" href="https://unpkg.com/spectre.css/dist/spectre-icons.min.css">
-  <style>
-    #advanced {
-      position: fixed;
-      bottom: 0;
-      left: 0;
-      width: 100vw;
-      border-top: 1px solid #F1F1F1;
-    }
-  </style>
+    <meta charset="utf-8">
+    <title>Scrim map generator</title>
+    <meta name="description" content="Generates map rotation for Destiny 2 scrims">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#fafafa">
+    <link rel="stylesheet" href="https://unpkg.com/spectre.css/dist/spectre.min.css">
+    <link rel="stylesheet" href="https://unpkg.com/spectre.css/dist/spectre-icons.min.css">
+    <link rel="stylesheet" href="styles/main.css">
 </head>
 <body>
-  <div class="container">
+<div class="container">
     <h2>
-      The following maps are selected
+        The following maps are selected
 
-      <button class="btn btn-link btn-action float-right" id="again">
-        <i class="icon icon-refresh"></i>
-      </button>
+        <button class="btn btn-link btn-action float-right" id="again">
+            <em class="icon icon-refresh"></em>
```

Comment:
Since this is an icon, `<em>` is not semantically correct. `<i>` is fine, or `<span>` to be a bit more correct.

Reply:
I see. I may have followed SonarLint's suggestions a bit too mindlessly.